hello-ironhack-react-app/src/components/InfoList.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import './InfoList.css'; // import the CSS
+import Info from './Info.js';
+
+function InfoList(props){
+    // assuming props is an array of Info objects
+    return(
+        <div className="info-list">
+            {props.infos.map((info)=>{
+                    return <Info image={info.image} title={info.title} text={info.text}/>;
```

Comment:
no need for return in this case. The codeblock will be executed as many times as (infos) contains elements.